webhook_to_fedora_messaging/endpoints/parser/parser.py

```diff
@@ -0,0 +1,63 @@
+
+from  flask import Request
+from werkzeug.datastructures.headers import EnvironHeaders
+import hashlib
+import hmac
+from webhook_to_fedora_messaging_messages.github.github import GithubMessageV1
```

Comment:
You need to add it as a dependency, with `poetry add webhook-to-fedora-messaging-messages`.